src/content/developers/docs/gas/index.md

```diff
@@ -74,7 +74,46 @@ For example if you put a gas limit of 50,000 for a simple ETH transfer, the EVM
 
 ## What is Gas price? {#what-is-gas-price}
 
-Gas price refers to the amount of Ether you are willing to pay for every unit of gas, and this is usually measured in 'gwei'.
+Gas price refers to the amount of Ether you are willing to pay for every unit of gas, and this is usually measured in 'gwei'. Prior to
+[the London update](history/#london), you specify in the transaction how much you are willing to pay per gas, and you pay exactly that amount. 
+Different transactions in the same block can have very different gas prices.
+
+### The London Update {#gas-price-london-update}
+
+Starting with the London update, every block has a base fee, the minimum per gas price for inclusion in this block. The base fee is calculated by a formula
+that compares the size of the previous block (the amount of gas used for all the transactions) with a target size. If the block size is higher than the target, there
+is more demand for inclusion in the blockchain than supply so the base fee, is increased. If the block size is lower than the target then there is more supply 
```

Comment:
> If the block size is higher than the target, there is more demand for inclusion in the blockchain than supply so the base fee, is increased.

Perhaps we should note somewhere in here that max block size doubles with this upgrade, and the "target" is 50% of this, allowing for dynamic block sizes. 

Consider changing the end of this sentence to  (note comma placement too)
"...there is more demand for inclusion in the blockchain than supply, so the base fee in the following block is increased."